dotcom-rendering/src/components/CardHeadline.tsx

```diff
@@ -48,177 +43,74 @@ type Props = {
 	isHighlights?: boolean;
 };
 
-/** These represent a new set of fonts. They are extra large font sizes that, as a group, are only used on headlines */
-const boostedFontStyles = ({ size }: { size: SmallHeadlineSize }) => {
-	switch (size) {
-		case 'ginormous':
-			return `${headlineMedium64}`;
-		case 'huge':
-			return `${headlineMedium50}`;
-		case 'large':
-			return `${headlineMedium42}`;
-		case 'medium':
-			return `${headlineMedium34}`;
-		case 'small':
-			return `${headlineMedium28}`;
-		case 'tiny':
-			return `${headlineMedium24}`;
-	}
+const headlineSize = {
+	xxxlarge: headlineMedium64,
+	xxlarge: headlineMedium50,
+	xlarge: headlineMedium42,
+	large: headlineMedium34,
+	medium: headlineMedium28,
+	small: headlineMedium24,
+	xsmall: headlineMedium20,
+	xxsmall: headlineMedium17,
+	xxxsmall: headlineMedium15,
+	tiny: headlineMedium14,
 };
 
-const fontStyles = ({
-	size,
-	boostedFontSizes,
-}: {
-	size: SmallHeadlineSize;
-	boostedFontSizes: boolean;
-}) => {
-	if (boostedFontSizes) return boostedFontStyles({ size });
+const textSansSize = {
+	xxxlarge: textSans20,
+	xxlarge: textSans20,
+	xlarge: textSans20,
+	large: textSans20,
+	medium: textSans20,
+	small: textSans20,
+	xsmall: textSans20,
+	xxsmall: textSans17,
+	xxxsmall: textSans15,
+	tiny: textSans12,
+};
```

Comment:
Labs dont currently have enough font sizes to match the type of headline. There will be a follow up PR to address labs styling